packages/organizations/origin-organization-irec-api/src/connection/connection.module.ts

```diff
@@ -3,23 +3,26 @@ import { CqrsModule } from '@nestjs/cqrs';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { ConfigModule } from '@nestjs/config';
 import { PassportModule } from '@nestjs/passport';
+import { ScheduleModule } from '@nestjs/schedule';
 
 import { RegistrationModule } from '../registration';
 import { IrecModule, IrecService } from '../irec';
 import { ConnectionController } from './connection.controller';
 import { Connection } from './connection.entity';
 import { ConnectionHandlers } from './handlers';
+import { RefreshAllTokensTask } from './cron';
 
 @Module({
     imports: [
+        ScheduleModule.forRoot(),
```

Comment:
Then if you apply this in root you don't have to import it here

Reply:
done